frontend/src/components/Header/Header.tsx

```diff
@@ -50,21 +73,73 @@ function Header({ organisation }: HeaderProps) {
     return (
         <AppBar position="static">
             <Toolbar sx={{ justifyContent: "space-between" }}>
-                <Link to="/">
-                    <Box
-                        height={{ xs: "2rem", md: "2.5rem" }}
-                        component="img"
-                        src={THEME_TO_IMAGE[organisation]}
-                        alt="Organisation logo"
-                    />
-                </Link>
-                {showDateAndTime && (
-                    <Typography variant="h4" component="p">
-                        {date}
-                    </Typography>
-                )}
+                <Tooltip followCursor={true} title="Return to home screen">
+                    <Link to="/">
+                        <Box sx={{ alignItems: "center", display: "flex" }}>
+                            <Box
+                                height={{ xs: "2rem", md: "2.5rem" }}
+                                component="img"
+                                src={THEME_TO_IMAGE[props.organisation]}
+                                alt="Organisation logo"
+                            />
+                            {props.isAuthenticated ?
+                                <IconButton
+                                    color="headerButtons"
+                                    sx={buttonSX}
+                                    size="medium"
+                                >
+                                    <HomeIcon fontSize="large" />
+                                </IconButton> : null
+                            }
+                        </Box>
+                    </Link>
+                </Tooltip>
+
+                <Box
+                    sx={{ alignItems: "center", display: "flex" }}
+                    id="rightDiv">
+                    <Tooltip title="Open documentation">
+                        <IconButton
+                            color="headerButtons"
+                            onClick={() => setOpenHelpModal(true)}
+                            sx={buttonSX}
+                        >
+                            <HelpIcon fontSize="large" />
+                        </IconButton>
+                    </Tooltip>
+                    {showDateAndTime && (
+                        <Typography variant="h4" component="p" paddingBottom={"0.25rem"}>
```

Comment:
The padding unaligns it from the middle. Compare the first image (without padding) with the second image (with padding).

![image](https://user-images.githubusercontent.com/36072821/226417209-8b660c86-d1a0-4d5d-a070-b2344e836226.png)

![image](https://user-images.githubusercontent.com/36072821/226417252-158667ad-ae14-4d12-b513-ea838a441a80.png)

Reply:
I added it because it looked unaligned on my computer monitor but looking now at my laptop it instead seems unaligned 🥲